Utilities/bootstrap

```diff
@@ -1006,21 +1075,31 @@ def main():
     # Compute the build paths.
     if platform.system() == 'Darwin':
         build_target = "x86_64-apple-macosx"
-    elif platform.system() == 'Linux':              
+    elif platform.system() == 'Linux':
         if platform.machine() == 'x86_64':
-            build_target = "x86_64-unknown-linux"
+            build_target = "x86_64-unknown-linux-gnu"
         elif platform.machine() == "i686":
-            build_target = "i686-unknown-linux"
+            build_target = "i686-unknown-linux-gnu"
         elif platform.machine() == 's390x':
-            build_target = "s390x-unknown-linux"
+            build_target = "s390x-unknown-linux-gnu"
         elif platform.machine() == 'ppc64le':
-            build_target = 'powerpc64le-unknown-linux'
+            build_target = 'powerpc64le-unknown-linux-gnu'
         elif platform.machine().startswith("armv7"):
+            # TODO(compnerd) support soft float
             build_target = 'armv7-unknown-linux-gnueabihf'
         elif platform.machine() == 'aarch64':
-            build_target = 'aarch64-unknown-linux'
+            build_target = 'aarch64-unknown-linux-gnu'
         else:
             raise SystemExit("ERROR: unsupported machine:",platform.machine())
+    elif platform.system() == 'Windows':
```

Comment:
Are you sure these will return what you expect? `platform.machine()` can't return "ARM64" on ARM64 Windows? Microsoft tends to use the spellings X86, X64 / AMD64, ARM, and ARM64.

Reply:
Yeah, I should use the MSFT spelling as well.